README.md

```diff
@@ -1,43 +1,141 @@
+- A target Cognite Data Fusion project[^1]
+- Identity Provider groups and service principals, see [Developer Toolkit Identity Provider Configuration](https://developer.cognite.com/sdks/toolkit/idp).
+
+
+## Step 1: Install the toolkit CLI 
+
+To install the `cdf-tk` tool, you need a working Python installation >=3.9 (recommended 3.11). It is available as a command-line tool, and `cdf-tk --help` will give you available options:
+
+```
+pip install cognite-toolkit
+cdf-tk --help
+```
+
+Follow instructions on [Developer Toolkit Quickstart](https://developer.cognite.com/sdks/toolkit/quickstart).
+
+
+## Step 2: Configure modules
+
+A **module** is a bundle of Cognite Data Fusion resources that are coupled together as logical units, for example data pipelines or the configuration of an application. For convenience, related modules are combined in _packages_. See [Developer Toolkit Templates](https://developer.cognite.com/sdks/toolkit/templates) for a more extensive description. 
+
+To install the modules, run this command with <my_project_dir> of your own choosing:
+
+```sh
+cdf-tk init <my_project_dir> 
+```
+
+
+### Select which modules to deploy
+
+Edit the `<my_project_dir>/local.yaml` file. You can add multiple environments (Cognite Data Fusion Projects[^1]) like this:
+
+```yaml
+# environment name
+dev:
+  project: <customer>-dev # the cdf project name
+  type: dev
+  deploy: # packages or modules to deploy to this environment
+    - cdf_demo_infield 
+    - cdf_apm_simple
+```
+
+The environment (dev in the example) is used by the cli tool as `--env <environment>` to determine which configurations to deploy.
 
-Below is an overview of the scope of what can be governed through using these templates:
 
-![Overview of project templates](./static/overview.png "Overview")
 
-## Quickstart
+### Configure each module as needed
 
-To install the `cdf-tk` tool, you need a working Python installation >=3.9 (recommended 3.11).
-Run: `pip install cognite-toolkit`
+Each module follow a similar pattern:
 
-The `cdf-tk` tool is available as a command line tool. Run `cdf-tk --help` to see the available commands.
+```
+./modules|common|examples|local_modules/<moduleA>/
+                                                |- data_models/
+                                                |- auth/
+                                                |- transformations/
+                                                |- raw/
+                                                |- time_series/
+                                                |- files/
+                                                default.config.yaml 
+                                                config.yaml
+```
 
-## For more information
+Create a copy of default.config.yaml to config.yaml and edit it according to the customer setup. In general, this is the only file that should be edited in the modules. The build command will replace placeholder values in the templates with these values.     
```

Comment:
I believe we supply a default config.yaml, right? So this will confuse some people :)